### ISSUE TITLE
Question about render result

### ISSUE THREAD

**Issue by gentleman-zhong** [city-super/Scaffold-GS#74]
Hi guys, when I was rendering the model, there was a significant difference in the rendering effect between the training set and the test set. What is the reason for this.

train render result:
![image](https://github.com/user-attachments/assets/bf4fcd76-9237-417d-86d2-559626b11af9)

test render result:
![image](https://github.com/user-attachments/assets/1c364f45-0c19-407b-849d-93d146142f3c)

**Comment by tongji-rkr** [comment]
Can you check it out in viewer?
